concepts/language-model.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model takes text input (usually a history) and outputs the next word or character.
+
+Example:
+- input: `the man is riding a ___`
+- output: `bike` 40%, `car`: 30%, `scooter`: 10%, `blue`: 0.5%, ...
+
+In the example this would mean that the article `bike` has a 40% probability of being the continuation and is the most likely.
```

Comment:
In this example, the word `bike` has a 40% probability of being the continuation and is the most likely.

Reply:
That's a better wording, changed.